docs/model_developers/project_creation.rst

```diff
@@ -61,8 +61,11 @@ For example, the ``main.cf`` below calls the print plugin from the std module.
 .. note::
     The std module is the only module that does not have to be imported explicitly.
 
-This example can be executed with ``inmanta compile``
+Before the project can be executed, the std module has to be added to the *requires* section in the ``project.yml`` file and the
+ module needs to be installed in the development venv. This is done by executing the following command in the project directory:
```

Comment:
Since `autostd=True` by default, shouldn't we have `std` in the requires for each module (in the module cookiecutter template)? Then this section should just explain `inmanta project install`.

Reply:
You are right. I will update this.